utils/appliance/__init__.py

```diff
@@ -1617,29 +1613,10 @@ def configuration_details(self):
         except KeyError:
             return None
 
-    def server_id(self):
-        try:
-            return self.configuration_details[2]
-        except TypeError:
-            return None
-
-    def server_region(self):
-        try:
-            return self.configuration_details[0]
-        except TypeError:
-            return None
-
-    def server_name(self):
-        try:
-            return self.configuration_details[1]
-        except TypeError:
-            return None
-
-    def server_zone_id(self):
-        try:
-            return self.configuration_details[3]
-        except TypeError:
-            return None
+    server_id = db_queries.config_detail_method(db_queries.ConfigDetailResult.server_id)
```

Comment:
Are these still related to the ```self```, I'm not sure they specifically are. I have already done some cleanup in this area and did plan to tackle this at some point potentially moving much of this stuff into utils.appliance and not necessarily out of it. Thoughts??